common/app/views/support/cleaner/AmpEmbedCleaner.scala

```diff
@@ -82,12 +82,34 @@ case class AmpEmbedCleaner(article: Article) extends HtmlCleaner {
 
   }
 
+  def cleanAmpInteractives(document: Document) = {
+
+    document.getElementsByClass("element-interactive").filter { element: Element =>
+      element.getElementsByTag("a").length !=0
+    }.foreach { interactive: Element =>
+      val link = interactive.getElementsByTag("a")
+      val linkToInteractive = link.first().attr("href")
+      val iframe = document.createElement("amp-iframe")
+      // AMP uses the height and width ratio, so exact values don't matter
+      // All interactives should resize to the correct height once they load
+      iframe.attr("width", "5")
```

Comment:
Note the comment above on line 93. 
> AMP uses the height and width ratio, so exact values don't matter. 

This is probably not clear enough though, so I will add a link to the docs. In amp when you use the layout responsive, you only need to give a ratio for the height and width. The width will automatically be 100% and the height will change based on that ratio.

Hopefully that makes more sense.